packages/core/types/src/internal/namespace.ts

```diff
@@ -28,10 +28,10 @@ export type Global = 'global';
 /**
  * Represents any namespace
  */
-export type Any = API | Plugin | Admin | Strapi | Global;
+export type AnyNamespace = API | Plugin | Admin | Strapi | Global;
```

Comment:
If this represents any namespace shouldn't include the Modules for example?

Reply:
Not sure what you're referring too?